todo_item_class.py

```diff
@@ -0,0 +1,18 @@
+from trello_config import Config
+class ToDoItem:
+    def __init__(self, id, title, status, description):
+        self.id = id
+        self.title = title
+        self.description = description
+        self.status = status
+
+    @classmethod
+    def fromTrelloCard(cls, card, idList):
```

Comment:
Class method names should probably be snake case: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
However this is a good use of `@classmethod'